venia-integration-tests/src/tests/e2eTests/cart/userCanAddItemFromCategory.spec.js

```diff
@@ -0,0 +1,58 @@
+import {
+    categoryPage as categoryPageFixtures,
+    graphqlMockedCalls as graphqlMockedCallsFixtures,
+    productPage as productPageFixtures
+} from '../../../fixtures';
+
+import { categoryPage as categoryPageActions } from '../../../actions';
+
+import { header as headerAssertions } from '../../../assertions';
+
+import { aliasMutation } from '../../../utils/graphql-test-utils';
+
+const { categorySweaters, categoryJewelry } = categoryPageFixtures;
+
+const { carinaCardigan, silverAmorBangleSet } = productPageFixtures;
+
+const { addProductToCartFromCategoryPage } = categoryPageActions;
+
+const { assertCartIsEmpty, assertCartTriggerCount } = headerAssertions;
+
+const {
+    getCategoriesCall,
+    getProductDetailForProductPageCall,
+    hitGraphqlPath
+} = graphqlMockedCallsFixtures;
+
+describe('Verify Cart actions', () => {
+    it("User shouldn't be able to add a CustomizableProduct from category page", () => {
+        cy.intercept('GET', getCategoriesCall).as('gqlGetCategoriesQuery');
+
+        cy.intercept('GET', getProductDetailForProductPageCall).as(
+            'gqlGetProductDetailForProductPageQuery'
+        );
+
+        cy.intercept('POST', hitGraphqlPath, req => {
+            aliasMutation(req, 'AddProductToCart');
+        });
+
+        cy.visit(categorySweaters);
+        cy.wait(['@gqlGetCategoriesQuery'], {
+            timeout: 60000
+        });
+        // This wait is needed as element default loaded with just name and Url suffix (.html ) would be added later.based on configurations.
+        cy.wait(5000);
```

Comment:
Isn't there any mutation or query we can wait for instead of waiting for a certain time?
Here for exemple, we're waiting for the Category Name to show before interacting with the page. It should work in your case too. Poke me if you need assistance.
https://github.com/magento/pwa-studio/blob/develop/venia-integration-tests/src/tests/e2eTests/category/verifySortAndPagination.spec.js#L43